custom_student_verification/locale/ar/LC_MESSAGES/django.po

```diff
@@ -30,7 +30,7 @@ msgstr "تم التحقق من طلبك!"
 msgid ""
 "Your last application was rejected because: %(reason)s but you can apply "
 "again"
-msgstr "تم رفض طلب التحقق للأسباب: %(reason) لكن لايزال بإمكانكم إرسال طلب تحقق جديد"
+msgstr "تم رفض طلب التحقق للأسباب: %(reason)s لكن لايزال بإمكانكم إرسال طلب تحقق جديد"
```

Comment:
@giovannicimolin  The "s" is for english plural which I think does NOT apply here

Reply:
@alqahtani The `%(reasons)s` is the python syntax for formatting the input from the variable `reasons` as a _string (s)_ on the final translations. 
Check the Django documentation here: https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#standard-translation (look for `For this reason, you should use named-string interpolation (e.g., %(day)s) instead of positional interpolation (e.g., %s or %d) whenever you have more than a single parameter. If you used positional interpolation, translations wouldn’t be able to reorder placeholder text.`).